Bootstrap-Policies/README.md

```diff
@@ -1,42 +1,96 @@
 
-A Working Group is established by first defining a charter that can be
-ratified by the TC. A charter is a *statement of purpose*, a
-*list of responsibilities* and a *list of initial membership*.
+Please read the [Lifecycle Document][lifecycle] for more 
+information.
 
-A Working Group needs 3 initial members. These should be individuals
-already undertaking the work described in the charter.
+## Establishing a new TLP or TLWG
 
-The list of responsibilities should be specific. Once established, these
-responsibilities are no longer governed by the TC and therefore should
-not be broad or subjective. The only recourse the TC has over the Working
-Group is to revoke the entire charter and take on the work previously
-done by the Working Group themselves.
+In general, your process will be as follows:
 
-If the responsibilities described in the charter are currently
-undertaken by another WG then the charter will additionally have to be
-ratified by that WG.
+	1. Have a great idea for a TLP or a TLWG
+	2. Find some like-minded individuals
+	3. Copy the relevant documents from this directory 
+		and modify as needed for your TLP or TLWG
+	4. Submit an application for your TLP or TLWG to enter TSC 
+		Incubation (this can be done in parallel with #3)
+	5. The TSC will work with you to get your application accepted
+		into Incubation
+	6. The TSC will work with you to get your project or working
+		group out of Incubation
+	7. Then your endeavor will be a Planetary Software Top-Level Project
+		or a Planetary Software Top-Level Working Group, congrats!
+
+Whether you use these example documents or not, the most important
+thing for your nascent TLP or TLWG is to establish a charter.  A
+charter is a *statement of purpose*, a *list of responsibilities*,
+and a *list of initial membership*.
+
+### Statement of Purpose
+
+What is the goal of your TLP or TLWG?
+
+
+### List of Responsibilities
+
+The list of responsibilities should be specific. Once established,
+these responsibilities should not change, and therefore should not
+be broad or subjective. 
+
+If the responsibilities described in the draft charter are currently
+undertaken by another Planetary Software TLP or TLWG, then a
+discussion is required between the existing entity and your new
+proto-TLP or proto-TLWG.  The result will either be that you will
+remove that responsibility from your list, or the existing entity
+will alter their charter to accomodate the change of responsibility.
+
+
+### List of Initial Membership
+
+Those like-minded individuals referenced in Item #2 above are 
+referred to as the Tehcnical Committee (TC) for the TLP or TLWG.
+
+A TC should have at least 3 initial members to start the work and
+get the application submitted to the Planetary Software TSC, but
+it could be fewer.  It will need 3 members in order to graduate
+from Incubation. These should be individuals already undertaking
+the work described in the charter.
```

Comment:
No, the first sentence says you *should* have three people, but it could be fewer.  I wanted to emphasize 'should be three' but allow for the possibility of less.  Your suggested sentence seems confusing to me!  Did you mean it to be more absolute and say 'not be accepted?'

Reply:
I do not like how the first and second sentences say you should have 3 people, but you don't need 3 people, but you will eventually. I would rather it say, you do not need 3 people immediately, but will need 3 people eventually. Here's some potential text

> If you have fewer than 3 members in your TC or Working Group, your TLP or TLWG can be accepted under incubation. In order for a TLP or TLWG to graduate out of incubation, though, the TC or Working Group must have at least 3 members.

Reply:
Okay, that language works for me.  Para replaced.